R.swift/Generators/ImageGenerator.swift

```diff
@@ -59,15 +59,23 @@ struct ImageGenerator: Generator {
         )
       }
 
-    let functions = (assetFolderImageFunctions + imageFunctions)
-      .groupUniquesAndDuplicates { $0.callName }
+    let allFunctions = assetFolderImageFunctions + imageFunctions
+    let groupedFunctions = allFunctions.groupBySwiftNames { $0.callName }
```

Comment:
`sanitizeSwiftName` should be idempotent so that shouldn't give any problems. :)